README.md

```diff
@@ -1 +1,46 @@
-# truex-ios-mobile-reference-app
+# Overview
+
+This project contains sample source code that demonstrates how to integrate the true[X]
+Ad Renderer in iOS. This document will step through the various pieces of code that make
+the integration work, so that the same basic ideas can be replicated in a real production app.
+
+This reference app covers the essential work. It assumes your app already have a working ad manager.
+~~For a more detailed integration guide, please refer to: https://github.com/socialvibe/truex-mobile-integrations/~~ (OUTDATED)
+
+# Access the true[X] Ad Renderer Library
+
+One can get the true[X] Ad Renderer either by
+[CocoaPods](https://github.com/socialvibe/cocoapod-specs) 
+```
+source 'https://github.com/socialvibe/cocoapod-specs.git'
+
+target 'your-app' do
+    pod 'TruexAdRenderer-iOS', '3.2.3'
+end
+```
+or direct download and have added to your project appropriately.
```

Comment:
Where are they directly downloading the project from? 🧐

Reply:
I would imagine we will send the pub the updated URL for the build?

Reply:
Hmmm. So we'd just link them the `.zip` file for the `.framework`? That's fair. In that case, can we just say something like "or request a direct download of our SDK from our Integrations team"